README.md

```diff
@@ -1 +1,41 @@
-# NY-API-SOM
+# NY-API-SOM (Source: https://developer.nytimes.com/)
+
+## Description
+This is a SOM created for NY movies API. What this does is test data types which are used throughout the API. 
```

Comment:
The SOM doesn't test the API, it enables us to efficiently test the data using RSpec. We are not just testing the data types.